sdk/tests/deqp/framework/common/tcuTexCompareVerifier.js

```diff
@@ -778,7 +831,53 @@ tcuTexCompareVerifier.isNearestMipmapLinearCompareResultValid = function(level0,
                                               fBounds,
                                               cmpReference,
                                               result) {
+    var isFixedPointDepth = tcuTexCompareVerifier.isFixedPointDepthTextureFormat(level0.getFormat());
+
+    var w0 = level0.getWidth();
+    var w1 = level1.getWidth();
+    var h0 = level0.getHeight();
+    var h1 = level1.getHeight();
+
+    var uBounds0 = tcuTexVerifierUtil.computeNonNormalizedCoordBounds(sampler.normalizedCoords, w0, coord[0], prec.coordBits[0], prec.uvwBits[0]);
+    var uBounds1 = tcuTexVerifierUtil.computeNonNormalizedCoordBounds(sampler.normalizedCoords, w1, coord[0], prec.coordBits[0], prec.uvwBits[0]);
+    var vBounds0 = tcuTexVerifierUtil.computeNonNormalizedCoordBounds(sampler.normalizedCoords, h0, coord[1], prec.coordBits[1], prec.uvwBits[1]);
+    var vBounds1 = tcuTexVerifierUtil.computeNonNormalizedCoordBounds(sampler.normalizedCoords, h1, coord[1], prec.coordBits[1], prec.uvwBits[1]);
+
+    var minI0 = Math.floor(uBounds0[0]);
+    var maxI0 = Math.floor(uBounds0[1]);
+    var minI1 = Math.floor(uBounds1[0]);
+    var maxI1 = Math.floor(uBounds1[1]);
+    var minJ0 = Math.floor(vBounds0[0]);
+    var maxJ0 = Math.floor(vBounds0[1]);
+    var minJ1 = Math.floor(vBounds1[0]);
+    var maxJ1 = Math.floor(vBounds1[1]);
+
+    for (var j0 = minJ0; j0 <= maxJ0; j0++) {
+        for (var i0 = minI0; i0 <= maxI0; i0++) {
+            var x0 = tcuTexVerifierUtil.wrap(sampler.wrapS, i0, w0);
+            var y0 = tcuTexVerifierUtil.wrap(sampler.wrapT, j0, h0);
+
+            // Derivated from C++ dEQP function lookupDepth()
+            // Since x0 and y0 are wrapped, here lookupDepth() returns the same result as getPixDepth()
+            var depth0 = level0.getPixDepth(x0, y0, coordZ);
+
+            for (var j1 = minJ1; j1 <= maxJ1; j1++) {
+                for (var i1 = minI1; i1 <= maxI1; i1++) {
+                    var x1 = tcuTexVerifierUtil.wrap(sampler.wrapS, i1, w1);
+                    var y1 = tcuTexVerifierUtil.wrap(sampler.wrapT, j1, h1);
 
+                    // Derivated from C++ dEQP function lookupDepth()
+                    // Since x1 and y1 are wrapped, here lookupDepth() returns the same result as getPixDepth()
+                    var depth1 = level1.getPixDepth(x1, y1, coordZ);
```

Comment:
Derivated from C++ dEQP function lookupDepth().
Since x0 and y0 are wrapped, here lookupDepth() returns the same result as getPixDepth().